NativeImage.jenkins

```diff
@@ -0,0 +1,141 @@
+            sh "cd lemminx && git pull && cd .. || git clone https://github.com/eclipse/lemminx.git"
+            sh """
+              if [ ! -f graalvm-linux.tar.gz ]; then
+                curl https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-${params.GRAALVM_VERSION}/graalvm-ce-java8-linux-amd64-${params.GRAALVM_VERSION}.tar.gz -L --output graalvm-linux.tar.gz
+                tar -xzf graalvm-linux.tar.gz
+                ./graalvm-ce-java8-${params.GRAALVM_VERSION}/bin/gu install native-image
+              fi
+            """
+            sh "cd lemminx && JAVA_HOME=../graalvm-ce-java8-${params.GRAALVM_VERSION} ./mvnw clean package -Dnative -DskipTests && cd .."
+            sh "cp lemminx/org.eclipse.lemminx/target/lemminx-linux* lemminx-linux"
+
+            stash name: 'lemminx-linux', includes: 'lemminx-linux'
+          }
+        }
+        stage("Windows native-image") {
+          agent {
+            label "win10"
+          }
+          steps {
+            powershell "if (Test-Path lemminx-win32.exe) { Remove-Item lemminx-win32.exe }"
+            powershell """
+              if (Test-Path lemminx) {
+                cd lemminx
+                git pull
+                cd ..
+              } else {
+                git clone https://github.com/eclipse/lemminx.git
+              }
+            """
+            powershell """
+              if (-not (Test-Path graalvm-windows.zip)) {
+                Invoke-WebRequest https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-${params.GRAALVM_VERSION}/graalvm-ce-java11-windows-amd64-${params.GRAALVM_VERSION}.zip -OutFile graalvm-windows.zip
+                Expand-Archive graalvm-windows.zip
+                .\\graalvm-windows\\graalvm-ce-java11-${params.GRAALVM_VERSION}\\bin\\gu install native-image
+              }
+            """
+            powershell """
+              Push-Location
+              Install-Module VSSetup -Scope CurrentUser
+              \$vs = Get-VSSetupInstance
+              if (\$vs -is [array]) {
+                \$vs_loc = (\$vs[0]).InstallationPath
+                \$vs_id = (\$vs[0]).InstanceId
+              } else {
+                \$vs_loc = (\$vs[0]).InstallationPath
+                \$vs_id = (\$vs[0]).InstanceId
+              }
+              Import-Module "\$vs_loc\\Common7\\Tools\\Microsoft.VisualStudio.DevShell.dll"
+              Enter-VsDevShell "\$vs_id" -DevCmdArguments "/arch=amd64"
+              Pop-Location
+              \$Env:JAVA_HOME = \"\$Env:WORKSPACE\\graalvm-windows\\graalvm-ce-java11-${params.GRAALVM_VERSION}\"
+              cd lemminx
+              .\\mvnw.cmd clean package -Dnative -DskipTests
+              cd ..
+            """
+            powershell "mv lemminx\\org.eclipse.lemminx\\target\\lemminx-windows*.exe lemminx-win32.exe"
+
+            stash name: 'lemminx-win32.exe', includes: 'lemminx-win32.exe'
+          }
+        }
+        // TODO: uncomment macOS related code
```

Comment:
why is this commented in the first place?

Reply:
So I can get to the next stage (packaging and uploading) when testing locally.